psydac/api/tests/test_postprocessing.py

```diff
@@ -47,6 +47,7 @@ def test_OutputManager():
 
     try:
         Om.export_fields(uh_static=uh)
+        raise ValueError
     except AssertionError:
         pass
```

Comment:
Is the purpose of this piece of code to check that an assertion is raised? If so, there is a specific way to do so with Pytest:

```suggestion
    with pytest.raises(AssertionError):
        Om.export_fields(uh_static=uh)
```
If the correct exception is not raised, the test will fail.